capitals.rb

```diff
@@ -150,3 +171,19 @@
     name: "Wyoming",
     capital: "Cheyenne"
 }]
+
+states = states.shuffle
+
+    while true
+      states.length.times do |state|
+        puts "What's the capital of " + states[state][:name] + "?"
+        response = gets.chomp
+        if response == states[state][:capital]
+          puts "Correct!"
+        else
+          response == false
+          puts "Sorry, that is incorrect"
+          break
+        end
+      end
+    end
```

Comment:
you could change this loop to break when user inputs that they don't want to play anymore.

```rb
until user_input == "no"
    user_input = gets.chomp
   #rest of code
end
```